setup.py

```diff
@@ -42,6 +42,7 @@ def run(self):
     scripts=[
         "rbd-target-gw.py"
     ],
+    data_files=[("/var/log/rbd-target-gw", [])],
```

Comment:
Should it be added to the spec file as well?

Reply:
Sorry. I missed this question.

We do not need to do a mkdir in the install section but we need a line in the files section:

%files
.....
%attr(3770,ceph,ceph) %dir /var/log/rbd-target-gw

and:

%attr(3770,ceph,ceph) %dir /var/log/rbd-target-api

in ceph-iscsi-cli

I am not sure what the permissions/user needs to be yet. I had just copied the values from the downstream ceph.spec when I was testing.

Checking now.

Reply:
Yeah, I was just asking if the spec needed to create the directory (not to copy that exact line).